time stamp
---
DetectedObjects detected_objects
bool success
